### PR TITLE
Add make msi

### LINKED ISSUE #2038: Release fails with make msi missing
It looks like #2001 was missing the `make msi` definition, e.g. [here](https://github.com/gopasspw/gopass/runs/4200291702?check_suite_focus=true).

### PR DESCRIPTION
**Pull request by dominikschulz**
Fixes #2038

Signed-off-by: Dominik Schulz <dominik.schulz@gauner.org>

### CODE REVIEW

**Comment by dominikschulz** [comment]
I gave it a quick try on Windows 10. It produces a valid MSI installer that (silently) installs gopass to the users AppDir.
Not sure if that's what Windows users expect, but it does work.

We can change it to a system install, but that will then require elevated privileges.

File: helpers/msipkg/main.go [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    <Media Id="1" Cabinet="app.cab" EmbedCab="yes"/>
+    <Icon Id="icon.ico" SourceFile="{{ .Icon }}"/>
+    <Property Id="ARPPRODUCTICON" Value="icon.ico"/>
+    <Directory Id="TARGETDIR" Name="SourceDir">
+      <Directory Id="LocalAppDataFolder">
+        <Directory Id="INSTALLDIR" Name="gopass">
+          <Component Id="gopass.exe" Guid="*">
+            <File Id="gopass.exe" Source="{{ .Binary }}" Name="gopass.exe"/>
+            <Shortcut Id="StartMenuShortcut" Advertise="no" Icon="icon.ico" Name="gopass" Directory="ProgramMenuFolder" WorkingDirectory="INSTALLDIR" Description=""/>
+            <Shortcut Id="DesktopShortcut" Advertise="no" Icon="icon.ico" Name="gopass" Directory="DesktopFolder" WorkingDirectory="INSTALLDIR" Description=""/>
```
Comment (AnomalRoil) [inline helpers/msipkg/main.go:36]: I don't see why we would have a start menu or a desktop shortcut for a cmd line app?

**Review by AnomalRoil — APPROVED** [review]
LGTM

File: helpers/msipkg/main.go (hunk 1 above)
Comment (dominikschulz) [reply helpers/msipkg/main.go:36]: For me it didn't show up in either place. IIUC thats the effect of `Advertise=no`. But I might be wrong.

**Pull request merged**